bc_obps/common/permissions.py

```diff
@@ -106,6 +106,11 @@ def get_permission_configs(permission: str) -> Optional[Union[Dict[str, List[str
         "authorized_irc_user": {
             'authorized_app_roles': AppRole.get_authorized_irc_roles(),
         },
+        "irc_director": {
```

Comment:
I called it IRC instead of CAS for consistency with the rest of the permission group names